src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -55,24 +55,24 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by the user, or empty list if user has never posted a
    *     message. List is sorted by time descending.
    */
-  public List<Message> getMessages(String user) {
+  public List<Message> getMessages(String recipient) {
```

Comment:
I don't understand this change.
>>Gets messages posted by a specific user.
This function is supposed to return the message for given user?.
Basically, what's the difference between user and recipient?

Reply:
I should have changed the @return description. This function was returning the messages posted by the user, and the user could only post on their own board. Now, this function returns all the messages for the account of the page that the user is viewing.

Reply:
sg. Thanks.